bucket/figma.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "101.6",
+    "description": "Figma connects everyone in the design process so teams can deliver better products, faster.",
+    "homepage": "https://www.figma.com/",
+    "license": {
+        "identifier": "Shareware",
+        "url": "https://www.figma.com/summary-of-policy/"
+    },
+    "url": "https://desktop.figma.com/win/FigmaSetup.exe",
+    "hash": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+    "bin": "FigmaSetup.exe",
+    "shortcuts": [
+        [
+            "FigmaSetup.exe",
+            "Figma"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.figma.com/downloads",
+        "regex": ">v([\\d.]+)\\s+\\("
+    },
+    "autoupdate": {
+        "url": "https://desktop.figma.com/win/FigmaSetup.exe"
```

Comment:
Does it not support different URLs for specific versions?

That sometimes helps with archiving.

Reply:
They just keep updating the setup file and reuploading it. Cygwin does this too.

Reply:
Yes but if there is a way to get versioned setups, it might make it easier for those who want to use older versions via scoop install <app>@<version>

A check should certainly be made once to find out if they provide versioned releases as well.